package.json

```diff
@@ -1,30 +1 @@
-{
```

Comment:
This should not be all deleted... all of this text should still be in the package.json